### PR TITLE
Support running cfn graph checks

### PR DESCRIPTION
**Pull request by YaaraVerner**
Added the support for running custom graph checks for CFN in CFN runner
Created a test for checking the CFN graph running.
Removed the "CHECKOV_CLOUDFORMATION_GRAPH" environment variable.
Created another yaml check in order to check the integration with cfngoat.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/cloudformation/graph/graph_runner/test_running_graph_checks.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import os
+import unittest
+from checkov.cloudformation.runner import Runner
+
+
+class TestRunningGraphChecks(unittest.TestCase):
+
+    def test_runner(self):
+        dir_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), f"resources")
```
Comment (Saarett) [inline tests/cloudformation/graph/graph_runner/test_running_graph_checks.py:9]: I think it could be beneficial if we add a similar test for JSON files too

**Comment by Saarett** [comment]
Overall looks good!

File: integration_tests/test_checkov_json_report.py [hunk 2]
```diff
@@ -58,6 +59,11 @@ def validate_json_quiet(self):
             self.assertFalse(data["results"]["passed_checks"])
             self.assertTrue(data["summary"])
 
+    def validate_check_in_report(self, report_path, check_id):
+        with open(report_path) as json_file:
+            data = json.load(json_file)[0]
+        assert any(check["check_id"] == check_id for check in data["results"]["failed_checks"] + data["results"]["passed_checks"])
```
Comment (gruebel) [inline integration_tests/test_checkov_json_report.py:65]: Using `itertools.chain` here results in not creating a new list with failed and passed checks.
```suggestion
        assert any(check["check_id"] == check_id for check in itertools.chain(data["results"]["failed_checks"], data["results"]["passed_checks"]))
```
and don't forget to add the import 😉

File: tests/cloudformation/graph/checks/resources/LambdaFunction/template.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
+Description: MSK Cluster with required properties.
```
Comment (gruebel) [inline tests/cloudformation/graph/checks/resources/LambdaFunction/template.yaml:1]: Some copy paste leftover 😄

**Review by Saarett — APPROVED** [review]

**Pull request merged**